### PR TITLE
Fix/change password bug

### PR DESCRIPTION
**Pull request by PavloJedi**

### CODE REVIEW

File: front-end/src/components/ResetPassword/ModalWindow.jsx [hunk 1]
```diff
@@ -196,11 +215,15 @@ const ModalWindow = ({ open, handleClose }) => {
...
-                  newPasswordAgain: enteredNewPasswordAgain,
-                })
+                savePassword(
+                  currentUser._id,
+                  {
+                    currentPassword: enteredCurrentPassword,
+                    newPassword: enteredNewPassword,
+                    newPasswordAgain: enteredNewPasswordAgain,
+                  },
+                  reset(),
```
Comment (andriydzhrvts) [inline front-end/src/components/ResetPassword/ModalWindow.jsx]: Function as a third param?

File: front-end/src/components/ResetPassword/ModalWindow.jsx [hunk 2]
```diff
@@ -14,11 +14,13 @@ import { CurrentUserContext } from 'context/AppProvider';
...
 import { REGEX_PASSWORD } from 'helpers/regex';
 import useInput from 'hooks/useInput';
+import Loader from 'components/common/Loader/Loader';
+import { withSnackbar } from 'components/withSnackbar/withSnackbar';
 
 const style = {
   position: 'absolute',
   top: '50%',
-  right: '416px',
+  left: '60%',
```
Comment (andriydzhrvts) [inline front-end/src/components/ResetPassword/ModalWindow.jsx]: Are you sure that 60% is aligning by center on different screen sizes?

File: front-end/src/components/ResetPassword/ModalWindow.jsx (hunk 1 above)
Comment (PavloJedi) [reply front-end/src/components/ResetPassword/ModalWindow.jsx]: pass it into right place

File: front-end/src/components/ResetPassword/ModalWindow.jsx (hunk 2 above)
Comment (PavloJedi) [reply front-end/src/components/ResetPassword/ModalWindow.jsx]: changed to be 50%, so at all screens modal will be on centre of the screen

**Review by andriydzhrvts — APPROVED** [review]

**Pull request merged**